language/resource.language.fr_fr/strings.po

```diff
@@ -1365,7 +1365,7 @@ msgstr "Nominé pour"
 #: /1080i/Includes_Labels.xml
 msgctxt "#31288"
 msgid "Explore now."
-msgstr "Explorez maintenant."
+msgstr " "
```

Comment:
You shouldn't attempt to use to translations to hack the display to remove elements.